src/stylesheets/components/_header.scss

```diff
@@ -1,6 +1,10 @@
-// Override the default 33% width on the logo in GOV.UK Frontend (prevents
-// unnecessary wrapping of "GOV.UK Design System" on smaller tablet / desktop
-// viewports)
 .app-header-logo {
+  // Override the default 33% width on the logo in GOV.UK Frontend (prevents
+  // unnecessary wrapping of "GOV.UK Design System" on smaller tablet / desktop
+  // viewports)
   width: auto;
+
+  // Override the right padding normally used to reserve space for the
+  // mobile menu toggle.
+  padding-right: 0;
```

Comment:
We can get rid of this once we update to 5.10.2, right?

Reply:
Yup, I think that's the padding being removed in 5.10.2. This PR will likely get merged after we update the site to 5.10.2 so we should remove this from the PR before we merge 😊